src/pages/audio-rewards-page/store/store.test.ts

```diff
@@ -281,6 +284,68 @@ describe('Rewards Page Sagas', () => {
       )
     })
 
+    it('should attempt retry if below max retries', () => {
+      const provideDelay = ({ fn }: any, next: any) =>
+        fn.name === 'delayP' ? null : next()
+      return (
+        expectSaga(saga)
+          .dispatch(
+            claimChallengeReward({
+              claim: testClaim,
+              retryOnFailure: true
+            })
+          )
+          .provide([
+            ...claimAsyncProvisions,
+            { call: provideDelay },
```

Comment:
just fyi I _think_ you should be able to do a matcher here with `[call.fn(delayP), null]` if you want to avoid the extra helper method